README.md

```diff
@@ -15,8 +15,35 @@
 
 # <a name="overview"></a>Overview
 
-Each directory in this repository contains an automation workflow template for a specific community-built integration for Orbit workspaces.
+Inside this repository are template YAML workflow files you can use to automate **3rd party integrations** with your Orbit workspace.
```

Comment:
In this Overview section I’d focus more on the Why, than the How, e.g.: “This repository helps you get running with adding third party integrations to Orbit, in minutes”